.github/workflows/ccpp.yml

```diff
@@ -4,14 +4,26 @@ on: [push]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-    
+    container:
+      image: ubuntu:19.10
```

Comment:
Yeah, this is required because Ubuntu used in github is not the latest, so I use the ubuntu container from DockerHub.